.github/workflows/admin-eslint.yml

```diff
@@ -7,20 +7,33 @@ on:
           required: true
           type: string
         shopwareVersion:
+          description: |
+            With the default `.auto` it tries to find a matching shopware version with the shopware-version action.
+
+            For example if the current ref is next-1234, it tries to find the ref next-1234 in shopware/shopware. If there's non it will use shopwareVersionFallback
```

Comment:
```suggestion
            With the default `.auto`, the workflow tries to find a shopware version using the shopware-version action.

            For example, if the current ref is next-1234, it will try to find the ref next-1234 in shopware/shopware. If there's no matching ref, it will use shopwareVersionFallback
```